api/src/main/java/org/openmrs/module/patientqueueing/api/dao/PatientQueueingDao.java

```diff
@@ -11,21 +11,84 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.hibernate.Criteria;
+import org.hibernate.criterion.Order;
+import org.hibernate.criterion.Restrictions;
+import org.openmrs.Location;
+import org.openmrs.Patient;
+import org.openmrs.Provider;
 import org.openmrs.api.db.hibernate.DbSession;
 import org.openmrs.api.db.hibernate.DbSessionFactory;
+import org.openmrs.module.patientqueueing.model.PatientQueue;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
+import java.util.Date;
+import java.util.List;
+
 @Repository("patientqueueing.PatientQueueingDao")
 public class PatientQueueingDao {
 	
-	protected final Log log = LogFactory.getLog(PatientQueueingDao.class);
+	private static final Log log = LogFactory.getLog(PatientQueueingDao.class);
 	
 	@Autowired
 	DbSessionFactory sessionFactory;
 	
 	public DbSession getSession() {
 		return sessionFactory.getCurrentSession();
 	}
+
+	/**
+	 * @see org.openmrs.module.patientqueueing.api.PatientQueueingService
```

Comment:
The `@see` annotation should point not only to the class, but also the method. Did you look at openmrs-core for examples?